src/js/components/CreateAccountForm.js

```diff
@@ -145,61 +157,71 @@ class CreateAccountForm extends Component {
       this.state.email
         ? false
         : true;
+
     return (
-      <Form className="CreateAccount-form">
-        <div className="CreateAccount-nameFields">
+      <div className="CreateAccount-form">
+        <Form id="CreateAccountForm">
+          <div className="CreateAccount-nameFields">
+            <FormFieldHelper
+              baseClass="CreateAccount"
+              idPrefix="create-account"
+              name="first name"
+              onChange={this.handleFirstNameChange}
+            />
+            <FormFieldHelper
+              baseClass="CreateAccount"
+              idPrefix="create-account"
+              name="last name"
+              onChange={this.handleLastNameChange}
+            />
+          </div>
           <FormFieldHelper
             baseClass="CreateAccount"
             idPrefix="create-account"
-            name="first name"
-            onChange={this.handleFirstNameChange}
+            name="email"
+            onChange={this.handleEmailChange}
           />
+          {errors.emailFormat.length > 0 ? (
```

Comment:
empty strings are falsy so you can shorten this to 
```suggestion
          {errors.emailFormat ? (
```